packages/client/modules/userDashboard/components/OrgTeamMembers/OrgTeamMembers.tsx

```diff
@@ -0,0 +1,124 @@
+import useMenu from '../../../../hooks/useMenu'
+import {OrgTeamMembersMenu} from './OrgTeamMembersMenu'
+import {useDialogState} from '../../../../ui/Dialog/useDialogState'
+import DeleteTeamDialogRoot from '../../../../components/DeleteTeamDialogRoot'
+
+interface Props {
+  queryRef: PreloadedQuery<OrgTeamMembersQuery>
+}
+
+const query = graphql`
+  query OrgTeamMembersQuery($teamId: ID!) {
+    viewer {
+      team(teamId: $teamId) {
+        ...ArchiveTeam_team
+        id
+        billingTier
+        isOrgAdmin
+        isViewerLead
+        name
+        orgId
+        teamMembers(sortBy: "preferredName") {
+          id
+          isNotRemoved
+          ...AddTeamMemberModal_teamMembers
+          ...OrgTeamMembersRow_teamMember
+        }
+        tier
+      }
+    }
+  }
+`
+
+export const OrgTeamMembers = (props: Props) => {
+  const {queryRef} = props
+  const data = usePreloadedQuery<OrgTeamMembersQuery>(query, queryRef)
+  const {viewer} = data
+  const {team} = viewer
+  const {togglePortal: toggleModal, closePortal: closeModal, modalPortal} = useModal()
+  const {togglePortal, originRef, menuPortal, menuProps} = useMenu(MenuPosition.UPPER_RIGHT)
+
+  const {
+    open: openDeleteTeamDialog,
+    close: closeDeleteTeamDialog,
+    isOpen: isDeleteTeamDialogOpened
+  } = useDialogState()
+
+  if (!team) return null
+  const {isViewerLead, isOrgAdmin: isViewerOrgAdmin, teamMembers} = team
+
+  return (
+    <div className='max-w-4xl pb-4'>
+      <div className='flex items-center justify-center py-1'>
+        <div className='flex items-center'>
+          <Button size='md' shape='circle' variant='ghost' asChild>
+            <Link to={`/me/${ORGANIZATIONS}/${team.orgId}/teams`}>
+              <ArrowBack />
+            </Link>
+          </Button>
+          <h1 className='flex-1 text-2xl font-semibold leading-7'>{team.name}</h1>
+        </div>
+
+        <div className='ml-auto flex gap-x-2'>
+          <Button
+            variant='secondary'
+            shape='pill'
+            className='text-md w-32 self-center py-2'
+            onClick={toggleModal}
+          >
+            Add member
+          </Button>
+          <Button shape='circle' variant='ghost' ref={originRef} onClick={togglePortal}>
+            <MoreVert />
+          </Button>
+        </div>
+      </div>
+
+      <div className='divide-y divide-slate-300 overflow-hidden rounded border border-slate-300 bg-white'>
+        <div className='bg-slate-100 px-4 py-2'>
+          <div className='flex w-full justify-between'>
+            <div className='flex items-center font-bold'>{teamMembers.length} Active</div>
+          </div>
+        </div>
+        {teamMembers.map((teamMember) => (
+          <OrgTeamMembersRow
+            isViewerLead={isViewerLead}
+            isViewerOrgAdmin={isViewerOrgAdmin}
+            teamMemberRef={teamMember}
+            key={teamMember.id}
+          />
+        ))}
+      </div>
+
+      {modalPortal(
```

Comment:
@nickoferrall I've noted a bug just now that needs resolving before this can be merged. If the `ORG_ADMIN` is not a member of the team, the mutation that creates the MassInvitation link will fail. I can see 4 options:

1. If the ORG_ADMIN clicks the link, the client just-in-time fires a mutation to join this team
2. We do the same thing, but on the server-side: if an ORG_ADMIN tries to create a massInvite token, they are joined to the team first
3. Instead of having the ORG_ADMIN generate the mass-invite token, we masquerade as the Team Lead (ick)
4. We remove the "Add Member" functionality all together

I am split between options #1 and #4 and would love a second opinion. What do you think?

Reply:
Why not just allow creating mass invitations from non-team members if they are an org admin? It could be irritating to join all those teams and getting the notifications and all.

Reply:
I think that handling everything on the server would be ideal as it's more secure and less error-prone, but I think we should go with option four. 

The PR is already a big 'un. Shipping this without the "Add Member" functionality will still add a lot of value, so I think we should work on "Add Member" in a separate PR.

Reply:
@nickoferrall I like your logic there. Ok, I will do that.

> Why not just allow creating mass invitations from non-team members if they are an org admin? It could be irritating to join all those teams and getting the notifications and all.

The underlying data model has the massInvitation belonging to a TeamMember. I suppose the fifth (and best) option would be to modify the massInvitation table to have invitations also be able to belong to non-TeamMembers (i.e. the ORG_ADMIN) as well

I'd probably be nice to get that table off of rethinkdb before making that modification